doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -5,6 +5,19 @@ replace 1.14.3 by the correct version.
 Release  Walkthrough
 ====================
 
+Update Release documentation
+----------------------------
+
+The files ``doc/changelog/1.14.3-changelog.rst`` and
+``doc/release/1.14.3-notes.rst`` should be updated to reflect the final list of
+changes and contributors. These lists can be generated as::
+
+    $ python tools/changelog.py $GITHUB v1.14.2..maintenance/1.14.x > doc/changelog/1.14.3-changelog.rst
+
+where ``GITHUB`` contains your github access token. The list should be copied
+to ``*-notes.rst``. The file ``doc/source/release.rst`` should also be updated
```

Comment:
Only for release updates. We no longer include the changelog in the release notes for releases as it adds many pages to the documentation. There was some discussion of that back when ...